src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
```

Comment:
```suggestion
  const { 
    name, 
    age, 
    sex, 
    partnerName, 
    isMarried,
  } = person;
```
It's better for readability in destructuring to write every object property on a separate line.